python/cutlass/op/gemm.py

```diff
@@ -626,7 +626,7 @@ def _verify_type_and_layout(self, tensor, ref_type, ref_layout, name):
 
     def run(self, A=None, B=None, C=None, D=None,
             alpha=None, beta=None, sync: bool = True, print_module: bool = False, visitor_args: dict = None,
-            stream = default_stream()) -> GemmArguments:
+            stream = None) -> GemmArguments:
```

Comment:
Not clear why the default_stream related changes are needed. Please update the PR description to explain.